app/models/manageiq/providers/ovirt/inventory/parser/infra_manager.rb

```diff
@@ -98,17 +99,32 @@ def storagedomains
       total       = free + used
       committed   = storagedomain.try(:committed).to_i
 
+      storage_domain_type = storagedomain.dig(:type, :downcase)
```

Comment:
This looks weird, is there a `:downcase` key in the `:type` hash?  I would have expected `storagedomain.dig(:type)&.downcase`

Reply:
I think I got it from the old code https://github.com/ManageIQ/manageiq-providers-ovirt/pull/621/files/8fe7666ebdd56608e2858ea353e21ad0372a3f82#diff-43e38e24277dd68083169041f77e7830443aab1372bb4e7c472e3ad596cefffcL105, but I think `storagedomain.dig(:type)&.downcase` would be better?

Reply:
ah yeah you're right, so `storagedomain` isn't a hash, so this isn't the standard dig
```
(byebug) storagedomain.method(:dig)
#<Method: OvirtSDK4::StorageDomain(OvirtSDK4::Type)#dig(*keys) /home/grare/adam/.gem/ruby/3.0.0/gems/ovirt-engine-sdk-4.4.1/lib/ovirtsdk4/type.rb:79>
```

So this does work:
```
(byebug) storagedomain.dig(:type)
"iso"
(byebug) storagedomain.dig(:type, :downcase)
"iso"
```

Carry on